### ISSUE TITLE
UTCDateTimeAttribute can't be set to null

### ISSUE THREAD

**Issue by jin09** [pynamodb/PynamoDB#824]
Any attribute is allowed to be set to NULL however setting `UTCDateTimeAttribute` breaks during `deserialize`

**Model:**

```python
class UsersTable(Model):
    class Meta:
        table_name = "users"

    user_id = UnicodeAttribute(hash_key=True, default_for_new=str(uuid.uuid4()))
    datetime_ref = UTCDateTimeAttribute(null=True)
```

If I set `datetime_ref` in any row to NULL and then If I try to get that row from dynamo using PynamoDB API then I get the following error:

```bash
File "/venv/lib/python3.8/site-packages/pynamodb/pagination.py", line 183, in __next__
    item = self._map_fn(item)
  File "/venv/lib/python3.8/site-packages/pynamodb/models.py", line 513, in from_raw_data
    attributes[attr_name] = attr.deserialize(attr.get_value(value))
  File "/venv/lib/python3.8/site-packages/pynamodb/attributes.py", line 607, in deserialize
    return datetime.strptime(value, DATETIME_FORMAT)
```

**Comment by ikonst** [comment]
> set datetime_ref in any row to NULL

How are you doing that? What do you mean by "NULL"? "NULL" DynamoDB data type?

**Comment by jpinner-lyft** [comment]
@jin09 this has been fixed in the v5.0.0b1 release:

```
>>> user_model = UsersTable.from_raw_data({'user_id': {'S': '57692287-f7a1-435b-b37c-ffe5bb2189e7'}})
>>> print(user_model.user_id)
57692287-f7a1-435b-b37c-ffe5bb2189e7
>>> print(user_model.datetime_ref)
None
>>>
>>> user_model = UsersTable.from_raw_data({'user_id': {'S': '57692287-f7a1-435b-b37c-ffe5bb2189e7'}, 'datetime_ref': {'NULL': True}})
>>> print(user_model.user_id)
57692287-f7a1-435b-b37c-ffe5bb2189e7
>>> print(user_model.datetime_ref)
None
```
